### ISSUE TITLE
SEO Friendly?

### ISSUE THREAD

**Issue by mweichert** [worseisbetter/galleria#175]
Hi there,

I'm trying to determine if galleria can be used in a graceful degration scenario, that is SEO-friendly. Given the example below, what would be your thoughts:

In Main Frame

<pre>
# Not displaying left-angled bracket, as it causes rendering issues in the comment
image src="1.gif"/>
image src="2.gif"/>
img src="3.gif"/>
iframe src="/gallery.php"></iframe>
</pre>

In iFrame

<pre>
<script type='text/javascript'>
 parent.hideImages(); // for each image tag, set 'display:none'
 var imgs =  parent.getImageSources(); // [1.gif, 2.gif]
 gallerydata = someFuncWhichGeneratesJSON(imgs);
$('.gallery').galleria({
  data_type: 'json',
  data_source: gallerydata
});
</script>
</pre>

In other words, in an html document, generate a list of imgs. If Javascript is enabled, include an iframe which displays galleria. IMO, this is an example of degrading javascript, and is SEO friendly. Others are cautioned whether hiding images with Javascript is bad, as search engines will penalize you - whether it's only used for JS browsers or not.

Anxious to hear your thoughts. Excellent work on the galleria plugin, btw. I'd love to contribute in some way. Is there a list of issues available for me to take a look at?

Thanks,
Mike
